packages/ui/src/views/connectors/connector-entity-form.tsx

```diff
@@ -96,6 +98,11 @@ export const ConnectorEntityForm = (props: ConnectorEntityFormProps): JSX.Elemen
             </EntityFormSectionLayout.Header>
             <EntityFormSectionLayout.Form>
               <RenderForm className="space-y-4" factory={inputComponentFactory} inputs={formDefinition} />
+              {apiError && (
+                <Alert.Container variant="destructive" className="my-8">
+                  <Alert.Description>{apiError?.toString()}</Alert.Description>
```

Comment:
`apiError` is already checked in the conditional `({apiError && ...})`, so `apiError?.toString()` can be simplified to `apiError.toString()`.

Reply:
fixed